src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -51,16 +50,6 @@
   private File datastorePath;
 
   @Override
-  public List<File> getAppYamls() {
-    return appYamls;
-  }
-
-  public void setAppYamls(List<File> appYamls) {
-    this.appYamls = appYamls;
-  }
-
-
-  @Override
   public List<File> getServices() {
```

Comment:
Might also be a good opportunity to turn these from File's (which they should have never been) into Path's.